### DISCUSSION TITLE
Why iOS isn't supported

### DISCUSSION THREAD

**Discussion by EchterAlsFake** [EchterAlsFake/Porn_Fetch#18] [General]
Maybe some of you are asking why iOS isn't supported...

### The base issue
It's not because I hate Apple, it just doesn't work. Compiling for iOS is impossible. I'd need to get past the App Store through some shady methods and I even can't compile it for iOS, because the Qt Wheels I use for the Android development only support Android and Qt currently doesn't plan to add support for iOS with it. 

### And Emulators like iSH with the CLI? 

Nope, doesn't work. I tried with iSH, but even after compiling Python 3.11 13 hours (yeah 13 hours) I got an issue saying "Bad system call" and I couldn't find a fix.

Even if I compile the application with Pyinstaller, I get "Bad system call" by just starting pyinstaller. I tried to fix it, I looked up forums, but nobody ever had this issue.

### Next Issue

The output path. On Android I can just request storage permissions with Termux and the CLI and copy videos to /storage/emulated/0/Download, but on iOS I have no idea, since iOS is closed as f*ck and I don't mainly use iOS.

If you find a way through all of this, you can tell me under this thread and I'll add support for it.

**Comment by QueenAng** [comment]
It's not supported because iOS sucks
